features/reports/reports.feature

```diff
@@ -140,3 +141,44 @@ Feature: Reports
         And I press "Create"
         Then I should still be on the report creation page
         And I should see "Report code cannot be blank"
+
+    Scenario: Getting the correct sum with exchange rate
+        Given the following zones are defined:
+            | name        | type    | members                 |
+            | Scandinavia | country | Norway, Sweden, Finland |
+            | France      | country | France                  |
+        And there are following shipping categories:
+            | code | name    |
+            | SC1  | Regular |
+            | SC2  | Heavy   |
+        And the following shipping methods exist:
+            | code | category | zone        | name |
+            | SM1  | Regular  | Scandinavia | DHL  |
+            | SM2  | Heavy    | France      | UPS  |
+        And the following products exist:
+            | name | price | sku |
+            | Mug  | 10    | 456 |
+            | Book | 22    | 948 |
+        And the following orders exist:
+            | customer                | shipment                 | address                                                           | currency | exchange_rate |
+            | sylius@example.com      | UPS, shipped, DTBHH380HG | Théophile Morel, 17 avenue Jean Portalis, 33000, Bordeaux, France | EUR      | 1.00000       |
+            | linustorvalds@linux.com | DHL, shipped, DTBHH380HH | Linus Torvalds, Väätäjänniementie 59, 00440, Helsinki, Finland    | USD      | 0.5           |
+            | sylius@example.com      | UPS, shipped, DTBHH380HI | Théophile Morel, 17 avenue Jean Portalis, 33000, Bordeaux, France | GBP      | 1.25          |
+        And order #000000001 has following items:
+            | product | quantity |
+            | Mug     | 2        |
+            | Book    | 1        |
+        And order #000000002 has following items:
+            | product | quantity |
+            | Mug     | 2        |
+            | Book    | 1        |
+        And order #000000003 has following items:
+            | product | quantity |
+            | Mug     | 2        |
+            | Book    | 1        |
+        And order #000000001 will be completed on "2010-01-01"
+        And order #000000002 will be completed on "2010-01-02"
+        And order #000000003 will be completed on "2010-01-02"
```

Comment:
Yes, I put it in this syntax because sylius does it as well, is not failing, I asked myself the same question when I saw the code in github